frontend/packages/dev-console/src/components/edit-application/EditApplicationPage.tsx

```diff
@@ -73,27 +76,20 @@ const EditApplicationPage: React.FunctionComponent<EditApplicationPageProps> = (
         kind: 'BuildConfig',
         isList: true,
         namespace,
-        selector: {
-          matchLabels: { [NAME_LABEL]: nameLabel },
-        },
+        name: nameLabel,
         optional: true,
       },
       [PipelineModel.id]: {
         kind: referenceForModel(PipelineModel),
         isList: true,
         namespace,
-        selector: {
-          matchLabels: { [NAME_LABEL]: nameLabel },
-        },
+        name: nameLabel,
         optional: true,
       },
       imageStream: {
```

Comment:
So it's not a single imageStream anymore. Should we rename this to `namespacedImageStreams` and the `imageStreams` below to `openshiftImageStreams` or will this affect to many code areas for this small fix?